server/middleware/isAdmin.js

```diff
@@ -0,0 +1,22 @@
+const { checkAdminQuery } = require('../database/quieres');
+const { verifyToken } = require('../utils');
+
+const isAdmin = async (req, res, next) => {
+  try {
+    const { token } = req.cookies;
+    if (!token) {
+      return res.status(400).json({ message: 'You are not authorized' });
+    }
+    const decoded = await verifyToken(token);
+
+    const rows = await checkAdminQuery(decoded.email);
+    if (!rows.length) {
+      return res.status(400).json({ message: 'You are not authorized' });
+    }
+    res.status(201).json({ message: 'You are Admin' });
+    return next();
```

Comment:
This is a middleware, you only need to do your job and then move to the next controller/middleware if everything is going right, and I think it would be helpful if we updated the request object with the admin email `req.email = decoded.email` and I think that since there's no login for the admin, you didn't really test the cookies 
```suggestion
    return next();
```